lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -162,5 +163,20 @@ sub parent_group_overview {
     $self->group_overview('JobGroupParents', 'main/parent_group_overview');
 }
 
+sub changelog {
+    my ($self) = @_;
+
+    my $changelog;
+    if(open(my $changelog_file, '<', '/usr/share/openqa/ChangeLog')) {
```

Comment:
When putting it under `public/Changelog` the user will be able to access the raw change log. As you can see in this pull request I intend to embed the change log in the website and apply some formatting. But I guess it is ok to have both variants accessible.

Maybe I should also make the path configurable since it depends on the way openQA is packaged?